torchtext/experimental/datasets/raw/text_classification.py

```diff
@@ -0,0 +1,250 @@
+import torch
+import io
+from torchtext.utils import download_from_url, extract_archive, unicode_csv_reader
+
+URLS = {
```

Comment:
In torchaudio there recently has been issues with corrupted downloads. Either within this PR or at least as a follow-up item we should look into md5 verification of downloads to make sure the user is actually getting the correct data.